packages/component/src/Styles/defaultStyleOptions.js

```diff
@@ -11,6 +11,7 @@ const PADDING_REGULAR = 10;
 const DEFAULT_OPTIONS = {
   // Color and paddings
   accent: DEFAULT_ACCENT,
+  activeActivityOutlineColor: DEFAULT_SUBTLE,
```

Comment:
what about transcript focus?

Reply:
Added a few after reading WAI-ARIA for the terms. Would like to share the official terminology to you:

- Visual keyboard indicator, is the focus indicator, regardless it's for active descendant or not;
- The verb "focus" is also used when highlighting the `aria-activedescendant`, so it's not "activate".